TUTORIALS.md

```diff
@@ -32,20 +32,23 @@ npm install @cds/core@<version> @clr/angular@<version> @clr/ui@<version>
 
 - Add Clarity Styles: This is done later by adding the UKIS Theme
 
-- Add the Clarity module to app.config.ts:
+- Add the Clarity module and others to app.config.ts:
 ```
-import { ApplicationConfig, importProvidersFrom } from '@angular/core';
-import { ClarityModule } from '@clr/angular'; 
-import { provideAnimations } from '@angular/platform-browser/animations'; 
+import { ApplicationConfig, importProvidersFrom, provideZoneChangeDetection } from '@angular/core';
+import { ClarityModule } from '@clr/angular';
+import { provideAnimations } from '@angular/platform-browser/animations';
 import { BrowserModule } from '@angular/platform-browser';
+import { provideRouter } from '@angular/router';
 
+import { routes } from './app.routes';
 
 export const appConfig: ApplicationConfig = {
   providers: [
-    ...
-    importProvidersFrom(BrowserModule, ClarityModule, ...),
-    provideAnimations() //Replacement for BrowserAnimationsModule
-  ]
+    provideZoneChangeDetection({ eventCoalescing: true }),
```

Comment:
These two are optional and not required for a basic application. 
Also, the command (`ng new project-tutorial-map --style=scss --standalone=true`) we use to create the application is not for a routing application, so we do not need the `provideRouter`




> provideZoneChangeDetection({ eventCoalescing: true }),
> provideRouter(routes),

Reply:
Thank you for the hint, I will remove these two.